### PR TITLE
Added the phase titles, as well as the titles of the boards that the …

### PR DESCRIPTION
**Pull request by aalokb**
…player is currently viewing.

### CODE REVIEW

File: src/main/java/views/ApplicationController/index.ftl.html [hunk 1]
```diff
@@ -4,11 +4,22 @@
 <body>
 <h1>The Battleship game</h1>
 
+<div id="titles_container">
```
Comment (kramerje) [inline src/main/java/views/ApplicationController/index.ftl.html]: Okay, so this container is basically a notification to the user that transitions between states of the game, right? It helps with learnability and memorability.

Comment (aalokb) [reply src/main/java/views/ApplicationController/index.ftl.html]: Yeah, exactly

Comment (kramerje) [reply src/main/java/views/ApplicationController/index.ftl.html]: Awesome.

File: src/main/java/assets/css/custom.css [hunk 2]
```diff
@@ -44,12 +59,14 @@ dynamic_display {
         border-spacing: 0px;
         border: 1px solid black;
     }
+
+/*axes rows*/
```
[the remark this answers is not in the document]
Comment (aalokb) [reply src/main/java/assets/css/custom.css]: no problem, hopefully now it makes sense...

File: src/main/java/assets/css/custom.css [hunk 3]
```diff
@@ -24,12 +26,25 @@ body {
...
 dynamic_display {
     display: inline-block;
     margin-bottom: 80px;
 }
+    #titles_container {
+        height: 50px;
+        margin-bottom: 10px;
+    }
+    #players_titles_container {
+            height: 50px;
```
[the remark this answers is not in the document]
Comment (aalokb) [reply src/main/java/assets/css/custom.css]: eh, i like it.

**Pull request merged**